tests/test_qasm_examples.py

```diff
@@ -1,14 +1,34 @@
 import os
 import re
 import pytest
+import numpy as np
 
 from qdata import parser
 
+def fix_braces(line):
+    """Helper function to place braces on separate lines"""
+    for b in ["{", "}"]:
+        all_lines = []
+        for l in line:
+            split_lines = l.split(b)
+            if len(split_lines) == 1:
+                new_line = split_lines
+            elif split_lines[1] == "":
+                new_line = [clean(split_lines[0]), b]
+            elif split_lines[0] == "":
+                new_line = [b, *fix_braces(split_lines[1:])]
+            else:
+                new_line = np.ravel([[ll, b] for ll in split_lines])[:-1]
+
+            all_lines.extend([clean(l) for l in new_line if clean(l) and clean(l) != ";"])
```

Comment:
Looks like a good place ot use the walrus operator :=

Reply:
Oh, true. Never gotten to really use it yet. :+1: